_config.yml

```diff
@@ -345,16 +345,34 @@ font:
 # Third Party Services Settings
 # ---------------------------------------------------------------
 
-# MathJax Support
-mathjax:
+# Math Equations Render Support
+math:
```

Comment:
So, there is braking changes in `math.mathjax.enable`. So, need to create new branch and merge this changes with new 6.0.2 release version.

Reply:
So, Is there anything I could do about it?

Reply:
U can:
1. Create new branch and call it something like `feature katex` and push it near `mater` branch. When will be release, i just merge this branch into master.
2. Do nothing, i'll rebase this pull into `feature katex` branch and then merge this with release. But if so, this branch will from me as author, but author is u.